example_config/jira-auth-hook.py

```
@@ -0,0 +1,215 @@
+import os
+import subprocess
+import sys
+from pathlib import Path
+
+SENTINEL_TOKEN_BLOB = (
+    "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA6OjowMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDBh"
+    "NmM2MzI1MzM1NGQxODBiNjkzYWFjYmRkZjlmYjA2YzFkMGI2NmE0MmQ4Mzc1NmJjM2U5ZjM5ODg4MzRh"
+    "MGZiM2EzYTRhMWY="
+)
+TOKEN_CMD_ENV = "HARNX_JIRA_TOKEN_CMD"
+DEFAULT_TOKEN_CMD = 'secret-tool lookup service acli username "jira:{current_profile}"'
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to: "DEFAULT_TOKEN_CMD"

(S105)